client/modules/workspace/src/AppsWizard/FormField.tsx

```diff
@@ -85,11 +122,111 @@ export const FormField: React.FC<{
         style={{ textAlign: 'left', marginBottom: description ? 0 : 16 }}
       >
         {type === 'select' ? (
-          <Select
-            {...props}
-            value={getValues(name)}
-            style={{ textAlign: 'left' }}
-          />
+          name === 'configuration.execSystemId' ? (
```

Comment:
The aim of this file component is to be as dynamic as possible. Could we try refactoring this out to it's own component? We can also try extending the `Select` element to accept an `after` property, which would be a type `React.FC`, where we can pass the queue status logic